### PR TITLE
feat: bash completion

### PR DESCRIPTION
**Pull request by stuartwdouglas**
fixes: #2707

### CODE REVIEW

File: frontend/cli/main.go [hunk 1]
```diff
@@ -30,27 +31,28 @@ type CLI struct {
...
+	Build       buildCmd                  `cmd:"" help:"Build all modules found in the specified directories."`
+	Box         boxCmd                    `cmd:"" help:"Build a self-contained Docker container for running a set of module."`
+	BoxRun      boxRunCmd                 `cmd:"" hidden:"" help:"Run FTL inside an ftl-in-a-box container"`
+	Deploy      deployCmd                 `cmd:"" help:"Build and deploy all modules found in the specified directories."`
+	Migrate     migrateCmd                `cmd:"" help:"Run a database migration, if required, based on the migration table."`
+	Download    downloadCmd               `cmd:"" help:"Download a deployment."`
+	Secret      secretCmd                 `cmd:"" help:"Manage secrets."`
+	Config      configCmd                 `cmd:"" help:"Manage configuration."`
+	Pubsub      pubsubCmd                 `cmd:"" help:"Manage pub/sub."`
+	Completion  kongcompletion.Completion `cmd:"" help:"Outputs shell code for initialising tab completions"`
```
Comment (alecthomas) [inline frontend/cli/main.go:55]:
```suggestion
	Completion  kongcompletion.Completion `cmd:"" help:"Outputs shell code for initialising tab completions."`
```

**Review by alecthomas — APPROVED** [review]
Glorious!

**Comment by alecthomas** [comment]
I forgot to note - we should somehow reuse the kongpletion completion for readline.

**Comment by stuartwdouglas** [comment]
I figured, I am investigating how to do it. We may also want to provide more advanced completers at some point for things like verb names etc

**Pull request merged**